src/OpenTK.GraphicsLibraryFramework/Enums/InputAction.cs

```diff
@@ -0,0 +1,18 @@
+﻿//
+// InputAction.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+namespace OpenToolkit.GraphicsLibraryFramework
+{
+    public enum InputAction
```

Comment:
Is this a duplicate of https://github.com/opentk/opentk/blob/5dbaa047def2c0ec319e46d79c6811e128ce1f68/src/OpenTK.Windowing.Common/Enums/InputAction.cs ?

Reply:
Yes. Both GLFW and Windowing.Common both need it, but they can't reference each other.

Reply:
Is there a reason that the KeyCallbacks are implemented differently:
Usng IsRepeat boolean variable to recognize repeat Action(calling OnKeyDown), while if the InputAction is Release calling OnKeyUp.
Would make sense for MouseEvents in the same way. Or Implement the KeyEvent similar to the MouseEvent as well, as they are internally treated the same as well.